sections/exchange/FooterCard/TradeSummaryCard/TradeSummaryCard.tsx

```diff
@@ -39,11 +34,7 @@ type TradeSummaryCardProps = {
 
 const TradeSummaryCard: FC<TradeSummaryCardProps> = ({
```

Comment:
Is it expected that we have fee reclamation live right now? Should these execute atomic swaps?

I executed a trade and see the fee reclamation footer:
<img width="622" alt="image" src="https://user-images.githubusercontent.com/10401554/188012645-c3c01c1c-17ad-41fe-aca3-30906b78490a.png">

Reply:
I don't see any changes here impacting that, seems I only removed unused props

Reply:
Right, this only happens on OP goerli but not mainnet so it must be related to something else. We should note it but not block anything here.